src/main/java/step3/model/Car.java

```diff
@@ -0,0 +1,34 @@
+package step3.model;
+
+import java.util.Random;
+
+public class Car {
+    private static final int INITIAL_COUNT = 0;
+    private static final int MOVE_MIN_NUMBER = 4;
+    private static final int RANDOM_MAX_NUMBER = 10;
+    private static final Random random = new Random();
+
+    private int moveCount;
+
+    public Car() {
+        this.moveCount = INITIAL_COUNT;
+    }
+
+    public int getMoveCount() {
+        return moveCount;
+    }
+
+    public void moveOrStop(){
+        move(getRandomValue());
+    }
+
+    public void move(int randomValue) {
+        if(randomValue >= MOVE_MIN_NUMBER){
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
IDE로 IntelliJ IDEA를 사용 중이시라면,  자동 포맷팅 단축키(Windows에서는 `Ctrl + Alt + L`, Mac에서는 `Command + Option + L`)를 활용해보시면 좋을 것 같아요.
```suggestion
        if (randomValue >= MOVE_MIN_NUMBER) {
```